R/getKey.R

```diff
@@ -0,0 +1,29 @@
+# getKey.R
+#
+# ==============================================================================
+# SIDE EFFECTS:
+# This script imports biogridr which uses the deprecated function xml2::xml_find_one().
+# ==============================================================================
+#
+#' Generates a unique key for BioGrid API access
+#'
+#' @param my.name A string of the user's full name separated by space(s)
+#' @param my.email A string matching a valid user email
+#' @param my.project A string specifying a short project name (no spaces)
+#' @return The unique key for user and project for data retrieval from BioGrid
+#' @import biogridr
+#' @examples
+#' myKey <- getKey("Nada Elnour", "nada.elnour@@mail.utoronto.ca", "SILGRESA")
```

Comment:
Don't ever post valid credentials anywhere on GitHub. Even if these are not valid, it's better to use dummy names and variables. See [section 2.6.7 in the rptPlus README](https://github.com/hyginn/rptPlus#267-a-secure-approach-to-credentials).